integration/setup/setup.go

```diff
@@ -92,3 +105,45 @@ func Resources(c *client.AzureClientSet, g *framework.Guest, h *framework.Host)
 
 	return nil
 }
+
+func installCredential(h *framework.Host) error {
+	o := func() error {
+		k8sClient := h.K8sClient()
+
+		k8sClient.CoreV1().Secrets(credentialNamespace).Delete(credentialName, &metav1.DeleteOptions{})
+
+		c := env.AzureGuestConfig()
```

Comment:
Why do we need to create a `client.AzureClientSetConfig ` here? I think this is confusing. It suggests creating a client. I'd just take raw values for the `Data map[string][]byte` from `os.Getenv`.